library/types/src/modules/URLRecode.rb

```diff
@@ -47,7 +47,7 @@ def escape(input, safe_chars)
       return nil if input.nil?
       input.gsub(/[^#{safe_chars}]/) do |unicode_char|
         escaped = ""
-        unicode_char.each_byte { |b| escaped << sprintf("%%%02x", b) }
+        unicode_char.each_byte { |b| escaped << format("%%%02x", b) }
```

Comment:
So this is the only non-whitespace change in the code... and it uses a different alias for the same method. OK!